Yosemite/Yosemite/Model/Storage/BlazeCampaign+ReadOnlyConvertible.swift

```diff
@@ -24,13 +24,13 @@ extension Storage.BlazeCampaign: ReadOnlyConvertible {
     public func toReadOnly() -> BlazeCampaign {
         BlazeCampaign(siteID: siteID,
                       campaignID: campaignID,
-                      productID: productID?.int64Value,
+                      productURL: "", // TODO: map the new attribute `productURL` here
```

Comment:
nit: Can we create a GH issue to track these storage layer changes and add the issue number to the TODO?

Reply:
Updated in bed4ed901465cc2f0b0eec808056b3cf776492bd.